src/core/libraries/network/net.cpp

```diff
@@ -703,46 +763,58 @@ int PS4_SYSV_ABI sceNetEpollCreate(const char* name, int flags) {
         return ORBIS_NET_ERROR_EINVAL;
     }
 
-    auto epoll = Common::Singleton<EpollTable>::Instance()->CreateHandle(name);
-
-    return epoll;
+    auto fd = FDTable::Instance()->CreateHandle();
```

Comment:
It seems that `sceNetEpoll<Create,Destroy,Control>` from source files has the need to be wrapped with `NetErrorHandler` lambda like
```suggestion
int PS4_SYSV_ABI sceNetEpollCreate(const char* name, int flags) {

...
    return NetErrorHandler([&] {
        auto fd = FDTable::Instance()->CreateHandle();
        auto* epoll = FDTable::Instance()->GetFile(fd);
        epoll->is_opened = true;
        epoll->type = Core::FileSys::FileType::Epoll;
        epoll->epoll = std::make_shared<Epoll>(name);
        epoll->m_guest_name = name;
        return fd;
    });
}
```
for example